pages.es/common/az-sshkey.md

```diff
@@ -0,0 +1,21 @@
+# az sshkey
+
+> Administra claves públicas SSH.
```

Comment:
Based on https://learn.microsoft.com/es-es/cli/azure/sshkey?view=azure-cli-latest
```suggestion
> Administra claves públicas SSH con máquinas virtuales.
```
To update on the English page:
> Manage ssh public key with virtual machines.